docs/api.md

```diff
@@ -56,4 +57,6 @@ Public utility functions used elsewhere in the package.
     utils.experiments.fix_broken_second_line
     utils.experiments.compare_result_file_resample
     utils.experiments.assign_gpu
-```
+    utils.experiments.assign_gpu
```

Comment:
Do not add duplicate entries to the API page, and remove any duplicate entries in the file.